AnnoyancesFilter/sections/popups.txt

```diff
@@ -126,6 +126,7 @@ xn----7sba7aachdbqfnhtigrl.xn--j1amh#$#html { overflow: visible!important; }
 backblaze.com##.ju_Con
 honeysanime.com##.buruburu-bee
 nhac.vn##.ft-scroll-content
+fantia.jp##.fanclub-btns > div.popup-bar + div.popup-bar
```

Comment:
`fantia.jp##.fanclub-btns > div.popup-bar:last-child` or `fantia.jp##.fanclub-btns > div.popup-bar.is-clone`.

Reply:
Personally I prefer the latter.